tutorials/tutorialdev.yml

```diff
@@ -0,0 +1,5 @@
+name: gsoc
+channels:
+  - conda-forge
+  - defaults
+prefix: C:\Users\HRITHIK\anaconda3\envs\gsoc\envs\gsoc
```

Comment:
this is  a constant on your system, is this needed for others?

Reply:
I actually created this yml env file purposefully to help in the creating of a new environment for anybody wanting to test my code. It's not needed later. (I guess I should remove this now.)